### ISSUE TITLE
Store Grab Type (RSS, Search, Interactive Search) and Automatically Import Interactive Search with Mismatched title

### ISSUE THREAD

**Issue by LuckyPuppy514** [Sonarr/Sonarr#5043]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

#4698
this issue had mentioned:
when manual grab the release rejected by sonarr with: Unknown Series
sonaar will import into serial 01 automatically and cover

#4826
this issue had mentioned:
when the release title don't match indexer's TVDB Id
sonaar import automatically 
> pay attention: the release didn't rejected by sonarr

and i found the solution:
[Fixed: Don't automatically import if release title doesn't match series title](https://github.com/stevietv/Sonarr/commit/4cefc1ed19db72296affdfc72cd1bfd0b234e244)
and i saw: 
```
if (series == null)
```
it is related to #4698 : Unknown Series
but commit message show that solved #4826

it confuse me: 
#4698 user think the release is right but rejected by sonarr
#4826 user think the release is wrong but accepted by sonarr
they are different

but it did solve #4698 
i found if exist S01EXX, sonarr will not import and cover and wait user deal
but still import into series 01 automatically if there don't exist S01EXX
![image](https://user-images.githubusercontent.com/53246532/171549805-3977435a-e8ae-4786-8973-d5cf097f9767.png)

by the way to #4826
i think it should not blame to sonarr, it should blame to why indexer provide wrong TVDB id.
if sonarr can't believe the indexer TVDB id, why should believe the title?
it could be optimized,  but can't be solve fully
and it is right to download and import the release accepted by sonarr
otherwise what should be download and import?

### Describe the solution you'd like

import automatically with episode and series which is searching when manual grab Unknown Series

> when user manual grab
> i think they know which episode and series they want
> and they think it is the right release but rejected by sonarr so they manual grab
> and the episode and series must be what they are searching

> it would be wonderfull if sonarr import automatically with episode and series which is searching
> instead of editting manually

### Describe alternatives you've considered

if there exist S0EXX
i have to wait download finished and manual edit the episode and series.

if no
i have to wait download finished and go to folder, move it to the right path and rename.

### Anything else?

no, thanks

**Comment by bakerboy448** [comment]
> if no
> i have to wait download finished and go to folder, move it to the right path and rename.

There's really no situation where this is ever needed.

Sonarr shows every release in queue and unmatched completed downloads can be manually imported directly from queue. Perhaps you have show unknown (Activity => Queue => Options) disabled?

for all other ad-hoc manual imports use wanted => manual import

-—

It makes no sense to blindly trust release information from searches and it is trivial to import poorly named releases. In fact typing this request and pulling the other GHI took more time then you'd probably spend importing several months or a year of releases.

**Comment by LuckyPuppy514** [comment]
anime rejected with: Unknown Series frequently.
why user manual grab the release rejected by sonarr?
i think they know what they are doing.

and now sonarr import automatically into S01 when user manual grab
why S01?
user may search S02/S03/other series

**Comment by angrycuban13** [comment]
You are essentially suggesting to **blindly** trust what the indexers send back. Thats not a reasonable approach - especially with anime.

**Comment by bakerboy448** [comment]
> anime rejected with: Unknown Series frequently.


That's a support issue and typically due to needing an alias. This is discussed in the FAQ and Search Troubleshooting article.

**Comment by LuckyPuppy514** [comment]
#4698
this issue had mentioned:
when manual grab the release rejected by sonarr with: Unknown Series
sonaar will import into serial 01 automatically and cover

**Comment by LuckyPuppy514** [comment]
i already mention it above

i don't let you blindly trust what the indexers send back
i hope you trust user, they know which series they want

if you think it is unwise
i think it is also unwise to import S01EXX blindly when i am searching S02EXX.

**Comment by bakerboy448** [comment]
> i don't let you blindly trust what the indexers send back
> i hope you trust user, they know which series they want

This logic is illogical. The user is making a decision based on what the indexer returns.

> this issue had mentioned:
when manual grab the release rejected by sonarr with: Unknown Series
> sonaar will import into serial 01 automatically and cover

There's no issue. This was a bug that was fixed as it had the same root cause as Markus mentioned in the other GHI.

**Comment by LuckyPuppy514** [comment]
1. when user searching S02EXX but all release rejected by sonarr: Unkown Series
they try to manually grab the release which they trust, 
why not import to S02EXX ?
if there other choice?
i am not let sonarr accept all unkown series
i wish sonarr to trust what user choose.

2. if there no S01EXX, sonarr still import S01 automatically 
docker version: 3.0.8-develop
![image](https://user-images.githubusercontent.com/53246532/171556399-e94587c6-e797-451f-89ae-8cdb062bbc4a.png)

**Comment by bakerboy448** [comment]
> if there no S01EXX, sonarr still import S01 automatically
> docker version: 3.0.8-develop

There is no sonarr version of 3.0.8-develop.

What version in system => status?

Provide trace logs of this matching please
https://wiki.servarr.com/sonarr/troubleshooting#asking-for-help

**Comment by angrycuban13** [comment]
When you manually search a whole season you'd still be blindly trusting whatever is returned by the indexer. 

However, if you do episode by episode I'd be inclined to agree with you - the end user selected a specific release (episode) based on indexer results and it should be auto imported.

**Comment by LuckyPuppy514** [comment]
i manually grab this release
![image](https://user-images.githubusercontent.com/53246532/171562693-ff9d7bb6-5c52-4551-a482-112b34e2a17d.png)

sonarr.debug.0.txt
you can see it import to season 1 automatically
![image](https://user-images.githubusercontent.com/53246532/171564704-181b6283-cf0e-4271-9772-e15263c54f84.png)

key word:
```
盾之勇者成名錄

The Rising of the Shield Hero

/mnt/truenas/video/anime/
```

[sonarr.debug.0.txt](https://github.com/Sonarr/Sonarr/files/8821203/sonarr.debug.0.txt)
[sonarr.trace.0.txt](https://github.com/Sonarr/Sonarr/files/8821204/sonarr.trace.0.txt)

**Comment by LuckyPuppy514** [comment]
![image](https://user-images.githubusercontent.com/53246532/171565427-388428ce-659f-415b-bc79-672b4ffcc679.png)

**Comment by bakerboy448** [comment]
> 
> [sonarr.debug.0.txt](https://github.com/Sonarr/Sonarr/files/8821203/sonarr.debug.0.txt)
> 
> [sonarr.trace.0.txt](https://github.com/Sonarr/Sonarr/files/8821204/sonarr.trace.0.txt)
> 


Trace logs do not have any import and the wiki explicitly states entire log files shouldn't be shared.  Read the wiki, pay attention to the giant blue box, follow the 6 steps and provide a trace log of the context of the mismatch and import.

**Comment by bakerboy448** [comment]
Those logs are not really helpful as they are missing the critical details before where you grabbed it from where sonarr is parsing the release title...

**Comment by LuckyPuppy514** [comment]
![image](https://user-images.githubusercontent.com/53246532/171662839-d9ed2f73-3e69-4eaa-9a71-8e60199cad35.png)
i am confused about a big chunk

could you tell me what start time and end time log should i provide?

**Comment by bakerboy448** [comment]
`[Lilith-Raws] Tate no Yuusha no Nariagari S02 - 09 [Baha][WEB-DL][1080p][AVC AAC][CHT][MP4]`
> 2022-06-02 23:03:51.2|Debug|ParsingService|Using absolute episode number 9 for: The Rising of the Shield Hero - TVDB: 1x09

not a bug. poorly named release and does not have a season alias, working as expected

https://thexem.info/xem/show/4564

`Tate no Yuusha no Nariagari S2` is not `Tate no Yuusha no Nariagari S02`

**Comment by markus101** [comment]
TLDR, need to store whether a release was grabbed via interactive search, if so assume the user knows what they’re doing and import it, even if it’s a title mismatch.
